diff -up ./addons/Makefile.include.am.orig ./addons/Makefile.include.am
--- ./addons/Makefile.include.am.orig	2014-12-12 15:59:56.000000000 +0100
+++ ./addons/Makefile.include.am	2014-12-25 11:39:10.429807574 +0100
@@ -2,7 +2,7 @@
 # Makefile include for XBMC PVR add-ons
 #
 
-INCLUDES        = -I. -I$(abs_top_srcdir)/xbmc -I$(abs_top_srcdir)/lib @HOST_INCLUDES@
+INCLUDES        = -I. -I/usr/include/kodi -I$(abs_top_srcdir)/xbmc -I$(abs_top_srcdir)/lib @HOST_INCLUDES@
 WARNINGS        = -Wall -Wextra -Wno-missing-field-initializers -Woverloaded-virtual -Wno-parentheses
 DEFINES         = @ARCH_DEFINES@ -DUSE_DEMUX -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS
 AM_CXXFLAGS     = -g -O2 -fPIC $(WARNINGS) $(DEFINES) @HOST_CXXFLAGS@
